next/app/page.js

```diff
@@ -0,0 +1,216 @@
+"use client";
+import React, { useRef } from "react";
+import { IKImage, ImageKitProvider, IKUpload, IKVideo } from "imagekit-next";
+
+const publicKey = "<YOUR_IMAGEKIT_PUBLIC_KEY>";
+const urlEndpoint = "<YOUR_IMAGEKIT_URL_ENDPOINT>";
```

Comment:
Move all config to a single file so that it's easier for user to provide the config.

Reply:
Done, moved to `.env.local`